src/applications/disability-benefits/526EZ/pages/primaryAddress.js

```diff
@@ -8,14 +8,20 @@ import PhoneNumberWidget from 'us-forms-system/lib/js/widgets/PhoneNumberWidget'
 
 import ReviewCardField from '../components/ReviewCardField';
 
-import { PrimaryAddressViewField, AddressDescription } from '../helpers';
+import {
+  PrimaryAddressViewField,
+  ForwardingAddressViewField,
+  contactInfoDescription,
+  phoneEmailViewField
```

Comment:
Maybe in a separate PR, we should rename this page to `contactInformation` since it's more than just the `primaryAddress` now (and indeed has been for a while now).

Reply:
Yeah, I was thinking about that. Going to hold off on doing this in a separate PR, I think.